### ISSUE TITLE
Posix TZ string

### ISSUE THREAD

**Issue by hartmamt** [chronotope/chrono-tz#140]
Is there any way to pass in a time zone location, for example:

Africa/El_Aaiun returns "<GMT>0"?

**Comment by djc** [comment]
I don't think we currently support that. I'm not very fluent in Go code, so not sure how it synthesizes this data from what's in the TZDB.

**Comment by pitdicker** [comment]
This crate compiles the time zone data into a large table of transitions for a fixed set of years. The data doesn't include a TZ string like TZif files have.
